code/mob/living/carbon/human/procs/update_icon.dm

```diff
@@ -1218,40 +1218,12 @@ var/list/update_body_limbs = list("r_arm" = "stump_arm_right", "l_arm" = "stump_
 					mutant_image.color = "#FFFFFF"
 				src.body_standing.overlays += mutant_image
 
-#if ASS_JAM
-	src.maptext_y = 32
-	src.maptext_width = 64
-	src.maptext_x = -16
-	health_update_queue |= src
-#endif
-
 	if (src.bioHolder)
 		src.bioHolder.OnMobDraw()
 	//Also forcing the updates since the overlays may have been modified on the images
 	src.UpdateOverlays(src.body_standing, "body", 1, 1)
 	src.UpdateOverlays(src.hands_standing, "hands", 1, 1)
 
-#if ASS_JAM //Oh neat apparently this has to do with cool maptext for your health, very neat. plz comment cool things like this so I know what all is on assjam!
-/mob/living/carbon/human/UpdateDamage()
```

Comment:
might be better if this is just commented out, for future use, it's actually quite neat?

Reply:
The exact same code is used right below the removed part (in the `#else` block) for the murderbox training dummy, I think having an additional commented out copy isn't necessary.